### PR TITLE
Update documentation to include GE file functionality

### PR DESCRIPTION
**Pull request by nw-duncan**
Minor changes to the documentation were made to introduce the additional GE scanner physiological file functionality. 

## Proposed Changes
That GE file functionality now exists is stated and some specific instructions were added to the "how to". 

The API section was also updated to include the ``load_gep`` function that loads in GE files.

## Change Type

- [ ] `bugfix` (+0.0.1)
- [ ] `minor` (+0.1.0)
- [ ] `major`  (+1.0.0)
- [ ] `refactoring` (no version update)
- [ ] `test` (no version update)
- [ ] `infrastructure` (no version update)
- [X] `documentation` (no version update)
- [ ] `other`

## Checklist before review

- [X] I added everything I wanted to add to this PR.
- [ ] \[Code or tests only\] I wrote/updated the necessary docstrings.
- [ ] \[Code or tests only\] I ran and passed tests locally.
- [ ] \[Documentation only\] I built the docs locally.
- [X] My contribution is harmonious with the rest of the code: I'm not introducing repetitions.
- [X] My code respects the adopted style, especially linting conventions.
- [X] The title of this PR is explanatory on its own, enough to be understood as part of a changelog.
- [ ] I added or indicated the right labels.

- [ ] I added information regarding the timeline of completion for this PR.

- [ ] Please, comment on my PR while it's a draft and give me feedback on the development!

### CODE REVIEW

**Comment by nw-duncan** [comment]
It looks like this same error is occurring for other pull requests too:

image

A file missing somewhere, apparently, but I can't find out much more than that to try and solve it.

File: docs/index.rst [hunk 1]
```diff
@@ -48,7 +48,9 @@ phys2bids
 It was born for AcqKnowledge files (BIOPAC), and at the moment it supports
 ``.acq`` files as well as ``.txt`` files obtained by labchart
```
Comment (smoia) [inline docs/index.rst:49]:
```suggestion
``.acq`` files as well as ``.txt`` and ``.mat`` files obtained by AcqKnowledge or Labchart
```

File: docs/index.rst [hunk 2]
```diff
@@ -46,9 +46,11 @@ phys2bids
 
 ``phys2bids`` is a python3 library meant to format physiological files in BIDS.
 It was born for AcqKnowledge files (BIOPAC), and at the moment it supports
-``.acq`` files as well as ``.txt`` files obtained by labchart
+``.acq`` files as well as ``.txt`` and ``.mat`` files obtained by AcqKnowledge or Labchart
 (ADInstruments).
-It doesn't support physiological files recorded with the MRI, as you can find a software for it `here <https://github.com/tarrlab/physio2bids>`_.
+
+
+At the moment, it only supports physiological files from GE MRI scanners. You can find software that will work with files from other MRI scanner types `here <https://github.com/tarrlab/physio2bids>`_.
```
Comment (smoia) [inline docs/index.rst:53]:
```suggestion
While we aim at supporting all MRI proprietary physiological file formats, at the moment it only supports GE MRI scanners. You can find software that will work with files from other MRI scanner types `here <https://github.com/tarrlab/physio2bids>`_.
```

**Review by smoia — APPROVED** [review]
LGTM!

**Comment by smoia** [comment]
Don't worry about the pre-commit "error", it is indeed due to a configuration file not being there. The reason is that we need to push in quite a lot of refactoring before we do add it.
Equally, don't worry about the failing on test 310. It's not relevant to your PR, and I'll fix it straight after this PR gets merged.

**Comment by smoia** [comment]
:rocket: PR was released in [`2.8.4`](https://github.com/physiopy/phys2bids/releases/tag/2.8.4) :rocket:

**Comment by nw-duncan** [comment]
Excellent!

**Pull request merged**